### PR TITLE
Misc changes

### PR DESCRIPTION
**Pull request by stweil**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -60,16 +60,16 @@ PIP_INSTALL = pip install
...
 # Install test python deps via pip
 deps-test:
-	$(PIP) install -U "pip>=19.0.0,!=20.3.2"
+	$(PIP) install -U pip
 	$(PIP) install -r requirements_test.txt
 
 # (Re)install the tool
 install:
-	$(PIP) install -U "pip>=19.0.0,!=20.3.2" wheel
+	$(PIP) install -U pip wheel
```
Comment (bertsky) [inline Makefile:72]: ditto

File: Dockerfile [hunk 2]
```diff
@@ -17,7 +17,8 @@ COPY ocrd_validators/ ./ocrd_validators
 COPY Makefile .
 COPY README.md .
 COPY LICENSE .
-RUN apt-get update && apt-get -y install --no-install-recommends \
+RUN echo 'APT::Install-Recommends "0"; APT::Install-Suggests "0";' >/etc/apt/apt.conf.d/ocr-d.conf
+RUN apt-get update && apt-get -y install \
```
Comment (bertsky) [inline Dockerfile:21]: Ah, good to know!

File: Makefile (hunk 1 above)
Comment (stweil) [reply Makefile:72]: ditto

File: Makefile [hunk 3]
```diff
@@ -60,16 +60,16 @@ PIP_INSTALL = pip install
 
 # Dependencies for deployment in an ubuntu/debian linux
 deps-ubuntu:
-	apt-get install -y python3 python3-pip python3-venv imagemagick
+	apt-get install -y python3-pip python3-venv imagemagick
 
 # Install test python deps via pip
 deps-test:
-	$(PIP) install -U "pip>=19.0.0,!=20.3.2"
+	$(PIP) install -U pip
```
[the remark this answers is not in the document]
Comment (bertsky) [reply Makefile:67]: Right.

**Review by bertsky — APPROVED** [review]

**Pull request merged**